src/core/reducers/addons.js

```diff
@@ -262,6 +263,9 @@ export function createInternalAddon(
     addon.isRestartRequired = apiAddon.current_version.files.some(
       (file) => !!file.is_restart_required
     );
+    addon.isWebExtension = apiAddon.current_version.files.some(
```

Comment:
This isn't quite right since you could upload a webextension for desktop and a non-webextension for Android. However, that's a really unrealistic use case and simply highlights how our model of multiple install files is broken. Thus, I think you should keep it as is. I just wanted to mention it. I think switching to `files.all(...)` would make my example case worse from a usability standpoint so don't do that.

Reply:
The old site did `if current_version.files[0].is_webextension == True` so definitely don't to that 😆

Reply:
Yes, I thought about this too but told myself "we assumed that for other computed props", let's be consistent..

Reply:
I don't think there are plans to improve the data model any time soon. Your approach is fine -- someone will file a bug for their bizarro edge case if not :)